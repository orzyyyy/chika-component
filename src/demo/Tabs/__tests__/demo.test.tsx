import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import TabsDemo from '..';

describe('TabsDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<TabsDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
