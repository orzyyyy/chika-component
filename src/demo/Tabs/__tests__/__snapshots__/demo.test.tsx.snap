// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsDemo render correctly 1`] = `
<div
  className="Tabs_demo"
>
  <Tabs
    config={Object {}}
    currentSelect={1}
    onClick={[Function]}
  >
    <div
      data-label="test19"
    >
      test1
    </div>
    <div
      data-label="test28"
    >
      test2
    </div>
  </Tabs>
</div>
`;
